### PR TITLE
Project Bootstrap

### PR DESCRIPTION
**Pull request by cvilla714**
##HTML,  CSS & Bootstrap: 

Project Using Bootstrap

This project consists of replicating the website Newsweek.

- We used Bootstrap version 4.5 for the most part
- The content can be viewed on small screen sizes and all the way up to extra-large displays
- We also worked on the positioning of the elements

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: INVALID

## :wave: :wave: Hi there Team!!

Before submitting a project, ensure the project is ready to be reviewed.

- [ ] This is how the site looks from the live link preview and the files uploaded. (see image below)

- [x] Create a [Professional README File](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) The actual one is not descriptive enough.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

image

File: css/style.css
Comment (teekaytech) [inline css/style.css]: You should stick to using class names for your CSS selectors. Using ID is not a good CSS practice. Kindly change the selector to class.

File: css/style.css
Comment (teekaytech) [inline css/style.css]: Always keep your code DRY. You have a lot of duplicate rules in the blocks above. Rules like `font-size, line-height, cursor, etc` in the above blocks should be grouped. Please group common rules to avoid repetition.

File: css/style.css
Comment (teekaytech) [inline css/style.css]: Apply the DRY rule to these blocks and other places where you have repetition of rules.

File: css/style.css
Comment (teekaytech) [inline css/style.css]: The `!important` rule is great, but you're advised against it. When overused, it can quickly mess up your stylings. Please find a way to make things work without this rule.

File: index.html
Comment (teekaytech) [inline index.html]: Please give the `alt` attribute value for this image.

File: index.html
Comment (teekaytech) [inline index.html]: Always give the value of the image tag's `alt` attribute

File: index.html
Comment (teekaytech) [inline index.html]: Using ... does not have any semantic meaning. Use words that describe the images when they are not rendered.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've really done an amazing job so far, most especially with the responsiveness :ok:. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_As the main point of this project is to use Bootstrap, I will focus on this in my review. You're asked to be more precise and to make sure that your page looks exactly the same as the source page.__

- [x] Naming your pull request `Develop` is not meaningful enough. Please try to use a meaningful pull request title.

- [x] Since you're expected to be more precise with this project, try to use the same colors and boldings (buttons, headings etc) as used in the original page.

- [x] You're expected to use bootstrap **everywhere**. Most of the rules in your CSS file can be achieved with some CSS classes. Please check out boostrap typography classes and use more of them.



- [x] Always adhere to the CSS good practices.
- You have a lot of repetitions in your CSS rules. Check the ones I commented on below and make necessary corrections.
- You overused the `!important` rule. There are better ways of implementing that feature. Refactor your code to avoid overusing of the rule.
- [Check this article] for a better understanding of CSS good practices.

- [x] The hambugger icon is missing on the navigation bar for small and medium screen sizes.

Source page: 


Your page: 


- [x] In the `OPINION` section, the title of the opinions and the authors are in bold format. Observe the different from the image below and make necessary adjustments.

Source page: 


Your page: 


- [x] Observe the color and boldness of your headings in the slideshow section (US, WORLD, BUSINESS, etc) and the horizontal line. The line comes before the headings, not after them as you have in your page.

Source page:


Your page:


- [x] Check your subscribe and newsletter sections in both small and medium screen sizes. See the image below and observe the difference.

Source page:


**On medium screens**
Your page: 


**On large screens**
Source page:

402
Your page:


- [x] On large and medium screens, you are having some unnecessary space after the footer section of your page.

Source page:


Your page:


**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: css/style.css
Comment (umairarshadbutt) [inline css/style.css:461]: Remove  `!important`

File: index.html
Comment (umairarshadbutt) [inline index.html]: remove inline-style

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove inline-style

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Update the README file as privious TSE described
- [ ] Remove inline-styling and `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)



### Navbar
- [x] On 768px there is no user icon in navbar, there is a **Login**. Please fix this issue

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://cvilla714.github.io/bootstrap/#)|
|:---:|:---:|
|source-page|clone|

### Footer
- [x] remove the extra padding

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://cvilla714.github.io/bootstrap/#)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: css/style.css
Comment (cvilla714) [reply css/style.css:461]: I cannot remove important from this particular media query because otherwise, the code won't work as expected

File: index.html
Comment (cvilla714) [reply index.html]: ok done

File: index.html
Comment (cvilla714) [reply index.html]: done

File: css/style.css
Comment (cvilla714) [reply css/style.css:461]: the important rule I was overusing it so as a requested change  of the previous code reviewer  I got rid of almost all of it, as you can see I was using it several times, now I only have used it once

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hi, @cvilla714 and @Huemac-Alfredo. 
Both of you have done well on this project. Let's make a few changes, and we will be done.
You might have broken a few codes while fixing the extra padding requested by the previous TSE

- [x] The newsletter image is missing at a breakpoint of 767px down. Please see the image below: 
newsletter

- [x] Remove the border-right from the footer menu as is removed from the original at a breakpoint of 768px down. Refer to the screenshot below:
newsweek_footer_pa

- [x] We have to make sure nothing is messed up, see the image below, fix the border-right line on the large screen as seen on the original. 
newsweek-footer-border

- [x] There's a lot of images that need to be made 100% as you transition to the 767px breakpoint, see the image illustration. The clone is the one on the right. 

newsweek_extra-space

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by acushlakoncept — APPROVED** [review]
# Status: PROJECT APPROVED  🎉 🚀

#### Hi, @cvilla714  and @Huemac-Alfredo  
**Congratulations**  🎉 🎉 🎉  on completing your project, and taking the time to implement the changes requested by the TSEs.  I really commend your effort in putting in such a great deal of work to complete this project. It will sure look good on your profile.

Feel free to merge when ready. I wish you all the best in your next project.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Pull request merged**
